djangocms_snippet/models.py

```diff
@@ -48,15 +62,18 @@ class SnippetPtr(CMSPlugin):
     # lowercase of the class name of this model.
     # https://github.com/divio/django-cms/issues/5030
     cmsplugin_ptr = models.OneToOneField(
-        CMSPlugin, related_name='djangocms_snippet_snippetptr',
-        parent_link=True)
+        CMSPlugin,
+        related_name='djangocms_snippet_snippetptr',
```

Comment:
as django < 1.8 support is dropped, change ``related_name='djangocms_snippet_snippetptr'`` to ``related_name='%(app_label)s_%(class)s'``